package.json

```diff
@@ -24,7 +23,11 @@
     "connect-flash": "~0.1.1",
     "connect-mongo": "~0.3.3",
     "console-stamp": "~0.1.1",
+
+    "es6-constant": "^1.0.2",
```

Comment:
why do we need this?, all major browsers support const keyword now
https://kangax.github.io/compat-table/es6/